docs/systemd-with-workers/README.md

```diff
@@ -65,3 +65,37 @@ systemctl restart matrix-synapse-worker@federation_reader.service
 systemctl enable matrix-synapse-worker@federation_writer.service
 systemctl restart matrix-synapse.target
 ```
+
+## Hardening
+
+**Optional:** If further hardening is desired, the file
+`override-hardened.conf` may be copied from
+`contrib/systemd/override-hardened.conf` in this repository to the location
+`/etc/systemd/system/matrix-synapse.service.d/override-hardened.conf` (the
+directory will have to be created). It enables certain sandboxing features in
+systemd to further secure the synapse service. You may read the comments to
+understand what the override file is doing. A similar override file at
+`contrib/systemd-with-workers/override-hardened-worker.conf` may be copied to
+`/etc/systemd/system/matrix-synapse-worker@.service.d/override-hardened-worker.conf`
+(this directory will also have to be created) in order to apply the same
+hardening options to any worker processes.
+
+Once these files have been copied to their appropriate locations, simply run
+the following commands to apply the hardening options. They will automatically
+be applied at every restart as long as the override files are present at the
+specified locations.
+
+```sh
+systemctl daemon-reload
+
+# Restart the master
+systemctl restart matrix-synapse.service
+
+# If using a worker-based setup, restart the workers too.
+# To restart a specific worker (eg. federation_reader):
+systemctl restart matrix-synapse-worker@federation_reader.service
```

Comment:
People should be able to simply `systemctl restart synapse.target` to restart the master and all configured worker processes, rather than restarting each one individually.

Reply:
I'll be honest I didn't know this could be done even though I run an instance of Synapse myself! It seems one learns something new at every turn. I mostly copied stuff from the section above mine in the document, as you might have been able to tell.

Regardless I've made the suggested change.